tabpy-server/tabpy_server/app/app.py

```diff
@@ -285,48 +287,18 @@ def _validate_cert_key_state(msg, cert_valid, key_valid):
             log_and_raise(err, RuntimeError)
 
     def _parse_pwd_file(self):
+        succeeded = True
         if ConfigParameters.TABPY_PWD_FILE not in self.settings:
             logger.info("Password file is not specified")
```

Comment:
should succeeded be set to false if there is  no password file specified?

Reply:
This function is called without checking if the parameter was set in config or not and returns the result of parsing. For the case when there's no password file we don't want to fail.

But you are right - this code here is confusing, I'll rework it.